web/router.ex

```diff
@@ -17,7 +17,10 @@ defmodule Pairmotron.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
-    resources "/users", UserController
+    resources "/users", UserController do
+      put "/activate", UserController, :activate, as: :activate
+      put "/deactivate", UserController, :deactivate, as: :deactivate
```

Comment:
This doesn't seem to follow RESTful type behavior. Why are 'activate' and 'deactivate' their own routes here?